deploy/template.yaml

```diff
@@ -1427,6 +1427,86 @@ Resources:
               Bool:
                 "aws:SecureTransport": true
 
+  #
+  # CloudWatch DashBoard
+  #
+
+  AMFCloudWatchDashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+      DashboardBody:
+        !Sub >- 
+        {
+              "start": "-PT6H",
+              "periodOverride": "inherit",
+              "widgets": [{
+                "height": 6,
+                "width": 12,
+                "y": 0,
+                "x": 0,
+                "type": "metric",
+                "properties": {
+                  "title": "API Gateway 4xx, 5xx Errors and Total Invocations",
+                  "region": "eu-west-2",
```

Comment:
Could interpolate this to ${AWS::Region} and should work as we're providing the !Sub function for entire dashboard body

Reply:
Please be sure to update all other region references thanks

Reply:
Edited thanks 👍